newZealand.html

```diff
@@ -57,7 +56,7 @@ <h2>Cities</h2>
                 <li>Lower Hutt</li>
                 <li>Dunedin</li>
                 <li>Palmerston North</li>
-                </ul>
+            </ul>
         </div>
         <div class="kiwi">
             <h2>Landmarks</h2>
```

Comment:
needs <ul> on next line, to complete the "unordered list" thought, before continuing.